modules/itk_siteimprove/js/itk-siteimprove.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Adds siteimprove to the current page.
+ */
+
+(function ($, Drupal) {
+  'use strict';
+
+  Drupal.behaviors.itkSiteimprove = {
+    attach: function (context, settings) {
+      var key = settings.itk_siteimprove.key;
+
+      if (key) {
+        (function () {
```

Comment:
Why this extra inner function that invoked at once?

Reply:
Javascript replicated from https://support.siteimprove.com/hc/en-gb/articles/206343703-Adding-Siteimprove-Analytics-JavaScript-to-your-website

Reply:
check